### PR TITLE
Add assertTrueAboutPromise() and assertFalseAboutPromise()

### PR DESCRIPTION
**Pull request by ace411**
Dear maintainer,

I kindly request that you add the `assertTrueAboutPromise()` and `assertFalseAboutPromise()` test assertions to the library as I believe they are helpful in performing a vastness of checks on values subsumed in promises.

### CODE REVIEW

File: tests/AboutPromiseTest.php [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+namespace seregazhuk\React\PromiseTesting\tests;
+
+use Exception;
+use React\Promise\Deferred;
+use seregazhuk\React\PromiseTesting\TestCase;
+
+final class AboutPromiseTest extends TestCase
+{
+	/** @test */
+	public function predicate_function_reveals_what_is_true_about_promise(): void
```
Comment (seregazhuk) [inline tests/AboutPromiseTest.php]: Looks like you used tabs instead of spaces here. Can you fix it, please?

**Review by seregazhuk — APPROVED** [review]
Thank you for the request. Looks great! 👍 Can you fix one issue, please? And I'll merge it.

File: tests/AboutPromiseTest.php (hunk 1 above)
Comment (ace411) [reply tests/AboutPromiseTest.php]: Yes, I will. Thanks for responding.

Comment (ace411) [reply tests/AboutPromiseTest.php]: Does this look better? https://github.com/seregazhuk/php-react-promise-testing/pull/21/commits/a5a2f7d532237f2e2f2068cb6217ca8199e111dd

Comment (seregazhuk) [reply tests/AboutPromiseTest.php]: Looks great! 🔥 There is a conflict with master branch. Can you fix it? and I'll merge the request.

**Pull request merged**